diff -c OI-org/defs.H OI/defs.H
*** OI-org/defs.H	Fri Mar 11 08:35:10 1994
--- OI/defs.H	Fri Mar 11 08:35:49 1994
***************
*** 506,512 ****
  
  /* callback function types */
  /* the following is a backward compatiblity kludge, to work around compilers that do not accept typedef of member function */
! #if !defined(_AIX) && !defined(ultrix) && !defined(__encotech_zCC) && !defined(__osf__)
  #define TYPEDEF_MEMFN
  #endif
  
--- 506,512 ----
  
  /* callback function types */
  /* the following is a backward compatiblity kludge, to work around compilers that do not accept typedef of member function */
! #if !defined(_AIX) && !defined(ultrix) && !defined(__encotech_zCC) && !defined(__osf__) && !defined(__GNUC)
  #define TYPEDEF_MEMFN
  #endif
  
