diff -r -c util-linux-1.6/pathnames.h util-linux-1.6-slackware/pathnames.h
*** util-linux-1.6/pathnames.h	Sun Nov 21 11:30:54 1993
--- util-linux-1.6-slackware/pathnames.h	Tue May  3 22:23:41 1994
***************
*** 20,34 ****
   */
  
  #ifndef ETCBIN
! #define ETCBIN			"/etc"
  #endif
  
  #ifndef WTMPPATH
! #define WTMPPATH		"/etc/wtmp"
  #endif
  
  #ifndef VARPATH
! #define VARPATH			"/usr"
  #endif
  
  #define _PATH_BSHELL    	"/bin/sh"
--- 20,34 ----
   */
  
  #ifndef ETCBIN
! #define ETCBIN			"/sbin"
  #endif
  
  #ifndef WTMPPATH
! #define WTMPPATH		"/var/adm/wtmp"
  #endif
  
  #ifndef VARPATH
! #define VARPATH			"/var"
  #endif
  
  #define _PATH_BSHELL    	"/bin/sh"
***************
*** 38,57 ****
  #define TTYTYPES        	"/etc/ttytype"
  #define SECURETTY       	"/etc/securetty"
  #define _PATH_UTMP      	"/etc/utmp"
! #define _PATH_WTMP      	WTMPPATH
  
  #define	_PATH_DEFPATH	        "/usr/local/bin:/bin:/usr/bin:."
! #define	_PATH_DEFPATH_ROOT	ETCBIN ":/bin:/usr/sbin:/usr/bin"
  #define	_PATH_HUSHLOGIN		".hushlogin"
! #define	_PATH_LASTLOG		VARPATH "/adm/lastlog"
! #define	_PATH_MAILDIR		VARPATH "/spool/mail"
  #define	_PATH_MOTDFILE		"/etc/motd"
  #define	_PATH_NOLOGIN		"/etc/nologin"
  
  #define _PATH_LOGIN		"/bin/login"
  #define _PATH_INITTAB		"/etc/inittab"
  #define _PATH_RC		"/etc/rc"
! #define _PATH_REBOOT		ETCBIN "/reboot"
  #define _PATH_SINGLE		"/etc/singleboot"
  #define _PATH_SECURE		"/etc/securesingle"
  
--- 38,57 ----
  #define TTYTYPES        	"/etc/ttytype"
  #define SECURETTY       	"/etc/securetty"
  #define _PATH_UTMP      	"/etc/utmp"
! #define _PATH_WTMP      	"/var/adm/wtmp"
  
  #define	_PATH_DEFPATH	        "/usr/local/bin:/bin:/usr/bin:."
! #define	_PATH_DEFPATH_ROOT	"/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/sbin:/usr/sbin"
  #define	_PATH_HUSHLOGIN		".hushlogin"
! #define	_PATH_LASTLOG		"/var/adm/lastlog"
! #define	_PATH_MAILDIR		"/var/spool/mail"
  #define	_PATH_MOTDFILE		"/etc/motd"
  #define	_PATH_NOLOGIN		"/etc/nologin"
  
  #define _PATH_LOGIN		"/bin/login"
  #define _PATH_INITTAB		"/etc/inittab"
  #define _PATH_RC		"/etc/rc"
! #define _PATH_REBOOT		"/sbin/reboot"
  #define _PATH_SINGLE		"/etc/singleboot"
  #define _PATH_SECURE		"/etc/securesingle"
  
